haskell/haskell.bzl

```diff
@@ -423,8 +423,16 @@ def _make_package(
         use_empty_lib: bool) -> Artifact:
     artifact_suffix = get_artifact_suffix(link_style, enable_profiling)
 
+    def strip_prefix_dir(path, prefix):
```

Comment:
makes sense, I checked if Buck2's prelude already had such a utility, but there doesn't seem to be one.